pkg/controller/pravega/constants.go

```diff
@@ -11,13 +11,16 @@
 package pravega
 
 const (
-	cacheVolumeName       = "cache"
-	cacheVolumeMountPoint = "/tmp/pravega/cache"
-	tier2FileMountPoint   = "/mnt/tier2"
-	tier2VolumeName       = "tier2"
-	segmentStoreKind      = "pravega-segmentstore"
-	tlsVolumeName         = "tls-secret"
-	tlsMountDir           = "/etc/secret-volume"
-	heapDumpName          = "heap-dump"
-	heapDumpDir           = "/tmp/dumpfile/heap"
+	cacheVolumeName        = "cache"
+	cacheVolumeMountPoint  = "/tmp/pravega/cache"
+	tier2FileMountPoint    = "/mnt/tier2"
+	tier2VolumeName        = "tier2"
+	segmentStoreKind       = "pravega-segmentstore"
+	tlsVolumeName          = "tls-secret"
+	tlsMountDir            = "/etc/secret-volume"
+	heapDumpName           = "heap-dump"
+	heapDumpDir            = "/tmp/dumpfile/heap"
+	authVolumeName         = "auth-secret"
+	authMountDir           = "/etc/auth-volume"
+	defaultTokenSigningKey = "secret"
```

Comment:
Is this setting a default token signing key? if yes, we shouldn't use a default value for this, but rather throw an error as part of input validation.

Reply:
This change just moves the hard coded value "secret" to a variable. The secret is set via pravega spec options there is currently no code that validates the options provided. I would want to change all of this as part of a separate PR.

Reply:
Sure, that sounds good. Could you please create an enhancement issue and share the link here.

Reply:
I don't think this change or the enhancement is relevant for discussion on this PR. 
Nevertheless, here is the link: https://github.com/pravega/pravega-operator/issues/251